src/tools/illink/src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -3,11 +3,12 @@
     <!-- Keep these in sync with ILLinkTasksAssembly in Microsoft.NET.ILLInk.Tasks.props. -->
     <!-- Keep the net#.0 TFM in sync with the Mono.Linker.csproj condition below. -->
     <!-- Keep the net#.0 TFM in sync with the code in LinkTask.cs in method ILLink.ILLinkPath. -->
-    <TargetFrameworks>$(NetCoreAppToolCurrent);net472</TargetFrameworks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <Description>MSBuild tasks for running the IL Linker</Description>
     <IsPackable>true</IsPackable>
-    <PackageId>Microsoft.NET.ILLink.Tasks</PackageId>
+    <!-- Removing the package id to be able to build the linker inside runtime 
```

Comment:
I don't understand this change

Reply:
There is a Microsoft.NET.ILLink.Tasks package included in the Versions.props and Version.Details.xml since this PackageId generates another package with the same name then the build system will believe that there is a cycle. Once we have live bits being feed into runtime we can delete the package in Versions.props and Version.Details.xml and uncomment the PackageId in this project. Opened https://github.com/dotnet/runtime/issues/78073 to track this